Microsoft.Toolkit.Uwp.UI.Controls/AdvancedCollectionViewSource/AdvancedCollectionViewSource.Events.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.AdvancedCollectionViewSource
```

Comment:
True. The built-in one is in .Xaml.Data - https://msdn.microsoft.com/library/windows/apps/br209833?f=255&MSPPError=-2147217396 - I'd move it to the .UI project under the Data namespace based on that.